### PR TITLE
Add Pyladies Panama chapter location info.

### PR DESCRIPTION
**Pull request by Kizuu**
# PyLadies Pull Request
_Please document the following. We will aim to be as timely as possible in processing your pull request, however as this is a volunteer staffed project anticipate at least a week for pull requests to start to be reviewed._ 

## Is this pull requesting adding or updating a [PyLadies chapter website](https://github.com/pyladies/pyladies#to-add-a-new-pyladies-location)? 
 

- [ ] Yes
- [x] No

## Have you confirmed the website builds locally? See [here](https://github.com/pyladies/pyladies#run-the-site-locally).

- [x] Yes
- [ ] No

##  Does your pull request pass the `netlify/pyladies/deploy-preview` check?

- [x] Yes
- [ ] No

# Pull Request Checklist

- [x] Fill out all sections above
- [x] Share pull request in PyLadies Slack channel #project-tech-infra

/cc @pyladies/tech-and-infra-admins

### CODE REVIEW

**Review by Kizuu — COMMENTED** [review]
I see that in Netlify the name of the chapter appears as "Panamaacute;".

**Review by willingc — APPROVED** [review]
Welcome! This PR looks good to me.

**Comment by willingc** [comment]
@Kizuu What would you like the name to be?

File: www/config.yml [hunk 1]
```diff
@@ -1332,6 +1332,15 @@ chapters:
     latitude: -33.45
     longitude: -70.666667
   name: Santiago, Chile
+- email: panama@pyladies.com
+  image: Pyladies_Panama.png
+  location:
+    latitude: 8.54
+    longitude: 80.78
+  name: Panamaacute;
```
Comment (willingc) [inline www/config.yml:1340]: @Kizuu Whatever you place here will display on the website card for the chapter. I would remove the trailing semicolon `:`

Comment (Kizuu) [reply www/config.yml:1340]: Yes! I think the same about removing the trailing semicolon, the name that I wanted to show is "Panamá".

Comment (willingc) [reply www/config.yml:1340]: Cool! Once you update the PR and it looks good, please @-mention me and I will be happy to merge. Thanks.

Comment (Kizuu) [reply www/config.yml]: @willingc Thank you so much!

**Review by willingc — APPROVED** [review]

**Comment by willingc** [comment]
@Kizuu This should be live in a few minutes. Congrats!

**Pull request merged**